funnel/models/event.py

```diff
@@ -136,21 +163,8 @@ def attendees_by_event(cls, event):
         return db.session.execute(stmt).fetchall()
 
     @classmethod
-    def make_from_dict(cls, participant_dict, space):
-        return Participant(
-            fullname=participant_dict.get('fullname'),
-            email=participant_dict.get('email'),
-            phone=participant_dict.get('phone'),
-            twitter=participant_dict.get('twitter'),
-            job_title=participant_dict.get('job_title'),
-            company=participant_dict.get('company'),
-            city=participant_dict.get('city'),
-            proposal_space=space
-        )
-
-    @classmethod
-    def update_from_dict(cls, updated_participant):
-        participant = Participant.query.filter_by(email=updated_participant.get('email')).first()
+    def update_from_dict(cls, space, updated_participant):
+        participant = cls.query.get(space, updated_participant.get('email'))
```

Comment:
Did you mean `cls.get`? `cls.query.get` uses SQLAlchemy's primary key-based getter which won't work with objects like `space`.